elastic/kibana
# [SecuritySolution] Fix index filters

## Summary
Relevant issues: 
https://github.com/elastic/kibana/pull/148681/files#r1104594568
https://github.com/elastic/kibana/pull/136407

Please enable feature flag: chartEmbeddablesEnabled

### Steps to verify:
1. Check the selected index patterns you have from the sourcerer
2. Inspect KPIs on host / network / users pages, make sure it matches the selected index patterns from the sourcerer
3. Click `...` button of each KPIs and `Open in Lens`, Inspect > request, and check the displayed index patterns matches index filters.

### Before: Indices were missing from the inspect panel:
<img width="2554" alt="Screenshot 2023-02-13 at 19 50 47" src="https://user-images.githubusercontent.com/6295984/218560779-0df2a1e6-072c-4ccc-8004-e92256fadd05.png">

Index patterns were missing from Lens inspect panel: 
<img width="1281" alt="Screenshot 2023-02-13 at 19 51 14" src="https://user-images.githubusercontent.com/6295984/218560799-476614a6-fda2-452d-a043-47723c1bf13a.png">

### After:
<img width="2554" alt="Screenshot 2023-02-13 at 19 57 46" src="https://user-images.githubusercontent.com/6295984/218561890-3ac6206f-ad64-46fe-88cd-ef11abce0651.png">
<img width="2560" alt="Screenshot 2023-02-13 at 19 58 11" src="https://user-images.githubusercontent.com/6295984/218561897-b9d5046d-335e-4309-be9f-28025605434d.png">



### Checklist

Delete any items that are not applicable to this PR.
